apiserver/facades/client/modelgeneration/interface.go

```diff
@@ -42,6 +44,7 @@ type Generation interface {
 // Application describes application state used by the model generation API.
 type Application interface {
 	UnitNames() ([]string, error)
+	CharmSettingsWithDelta(settings.ItemChanges) (*state.Settings, error)
```

Comment:
I'm not sure you are going to need this either.

Reply:
Gone.